apps/fhir/bluebutton/views/read.py

```diff
@@ -56,7 +56,7 @@ def __init__(self, version=1):
         self.resource_type = "Patient"
 
     def get(self, request, *args, **kwargs):
-        return_c4dic = True
+        return_c4dic = False
```

Comment:
As is, this seems to always be false. Are we just enforcing this can't work at all for now?

Reply:
Wanted to leave this logic in there in case we wanted to hard code resources again, it would just be flipping these to true for whichever calls you wanted hard coded resources for and then adjusting the contents of the corresponding json file. Just to allow us to test with different resource fields without necessarily waiting for BFD.

Reply:
Perfect. I think that's fine to have to switch these manually to test. No world where we'd want that to work for anyone else, lol. Too ephemeral for a feature flag, otherwise